torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -5200,6 +5200,19 @@ Tensor lu_factor_ex_jvp(
   }
 }
 
+Tensor logsumexp_jvp(const Tensor& self_p, const Tensor& self_t, IntArrayRef dim, bool keepdim) {
+  auto self_p_exp = self_p.exp();
+  auto sumexp_p = self_p_exp.sum(dim, keepdim);
+
+  if (areAnyTensorSubclassLike({self_p, self_t}) || self_t._is_zerotensor()) {
```

Comment:
Are all the ops below supported by zero tensor efficiently?

Reply:
Not currently, according to fb.quip.com/Dv5hARK0xyTo, the missing ones are `sum` and `div`. 

Actually... we shouldn't need to worry about zero tensors for ops like this (that have only a single differentiable input) since we would be skipping the forward AD logic if the tensor does not have forward grad. The only worry though is if we have other formulas that reuse the jvp, for example softmax and logsoftmax here, but we are still OK for this PR because they also only have a single differentiable input.

Reply:
In that case we can remove the check?